com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -273,8 +273,7 @@ public static bool DrawHeaderFoldout(GUIContent title, bool state, bool isBoxed
             var menuRect = new Rect(labelRect.xMax + 3f, labelRect.y + 1f, 16, 16);
 
             // Add context menu for "Additional Properties"
-            Action<Vector2> contextAction = null;
-            if (hasMoreOptions != null)
+            if (contextAction == null && hasMoreOptions != null)
```

Comment:
Instead of a replacement, shouldn't it be additive?

Reply:
@alex-vazquez it is your code base, so what do you think?

Reply:
For being additive we will need another way of defining this context menus, I think is a good idea but we can potentially open a chain of changes that are not for this PR. Can you create a JIRA, and we can tackle it later.